cms/djangoapps/contentstore/views/videos.py

```diff
@@ -275,6 +278,20 @@ def video_images_upload_enabled(request):
     return JsonResponse({'allowThumbnailUpload': True})
 
 
+@ensure_valid_course_key
+@login_required
+@require_GET
+def video_sharing_enabled(request, course_key_string):
```

Comment:
I am wondering if we should make this endpoint a bit more generic than just `video_sharing`. 
Maybe have the code stay in the videos.py file, but create a functional to retrieve editing feature flags? Like `get_authoring_features`?